nx-X11/programs/Xserver/hw/nxagent/Reconnect.c

```diff
@@ -646,6 +647,10 @@ Bool nxagentReconnectSession(void)
     goto nxagentReconnectError;
   }
 
+  /* Re-read keystrokes definitions in case the keystrokes file has
+     changed while being supended */
+  nxagentInitKeystrokes(True);
+
```

Comment:
Does this also handle the following situation gracefully: before suspend, only system-wide keystrokes.cfg in place, after resumption, keystrokes.cfg in $HOME? From the code study, it should, shouldn't it?